docs/classes/ERC725.md

```diff
@@ -85,26 +93,28 @@ const decodedDataManyKeys = erc725.decodeData({
 erc725.encodeData(data);
 ```
 
-To be able to store your data on the blockchain, you need to encode it according to your `ERC725JSONSchema`.
-
+The function `encodeData` helps you to encode the data of a smart contract according to your `ERC725JSONSchema` so that you can store the information in smart contracts.
 :::tip
-When encoding JSON it is possible to pass in the JSON object and the URL where it is available publicly.
-The JSON will be hashed with `keccak256`.
+When encoding JSON, it is possible to pass in the JSON object and the URL where it is available publicly. The JSON will be hashed with `keccak256`.
 :::
 
 #### Parameters
 
-1. `data` - `Object`: An object with one or many properties, containing the data that needs to be encoded.
+| Name   | Type   | Description                                                                         |
+| :----- | :----- | :---------------------------------------------------------------------------------- |
+| `data` | Object | An object with one or many properties containing the data that needs to be encoded. |
 
 #### Returns
 
-`Object`
+| Name          | Type   | Description                                                                                     |
+| :------------ | :----- | :---------------------------------------------------------------------------------------------- |
+| `encodedData` | Object | an object with the same keys as the object passed in as a parameter containing the encoded data |
 
-An object with the same keys as the object that was passed in as a parameter containing the encoded data, ready to be stored on the blockchain.
+After the `data` is encoded, the object is ready to be stored in smart contracts.
 
-#### Example
+#### Single-Key Example V1
```

Comment:
I'd probably remove the `v1` here or just go with `Single-Key Example 1`. It could get confusing as it could be interpreted as a different version of the library